run_uh.py

```diff
@@ -121,22 +121,18 @@ def excepthook(exception_type, value, tb):
 		with open(outfilename, 'a') as f:
 			traceback.print_exception(exception_type, value, tb, file=f)
 		traceback.print_exception(exception_type, value, tb)
-		print('')
-		print(T('Unknown Horizons has crashed.'))
-		print('')
+		print('\n', T('Unknown Horizons has crashed.', '\n'))
```

Comment:
I suppose that's a matter of preference, in the current code it's quite obvious where new lines are and how this will look like in the terminal.